include/tc/core/execution_engine.h

```diff
@@ -72,10 +72,15 @@ class ExecutionEngine {
       std::function<bool(const ExecutorType*)> pruningFunction =
           [](const ExecutorType*) { return false; });
 
-  /// "Low-latency" execution mode in which we just propagate raw pointers to
-  /// data in GPU address space.
-  /// No tensor-related information can be checked so it is the user's
-  /// responsibility to ensure that shapes and strides match.
+  typename ExecutorType::ProfilingInfoType profile(
```

Comment:
This is likely going to be non-core team facing API, please document the function